slides/src/content/a11y.md

```diff
@@ -0,0 +1,166 @@
+
+1. Content and functionality that cannot be operated through a keyboard
+1. Form fields that are not labelled properly
+1. Images that don’t have alternative text
+1. Insufficient contrast between text and background
+1. Reading order that is not in a meaningful sequence
+
+Designers and developers can include several accessibility and design features to overcome challenges:
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Design - Font and Colour
+
+- Fonts: Use sans serif fonts and set a base pixel measure at 1em = 16px (=12pt)
+  - Alter the base depending on the device type: Desktop = 1.0em, Tablet = 1.1em, Smartphone=1.2em
+- Colour:  Ensure foreground:background ratios are 4:5:1 for normal text and 3:1 for large text
+- Images: Images must be alt tagged or have figures/numbers describing the image
+<br><br>![alt text](content/images/a11yfont.png "example of accessible font and colour ratio") ![alt text](content/images/a11ycolour2.png "example of accessible font and colour ratio")
+<br>Two images showing samples of a accessible font types and fore/back ground colour ratios
+<br>
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Design - Tables
+- Tables:  Do not use tables for layout, only data, and explicitly label columns and rows
+<br><br>![alt text](content/images/a11yinactab.png "example of inaccessible data table") 
+<br>Example of an inaccessible table, cells span multiple rows and columns
+<br>
+<br><br>![alt text](content/images/a11yactab.png "example of accessible table")
+<br>Example of an accessible table, all rows/columns explicitly labelled
+<br>
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Design - Links and Buttons
+- Links and Buttons: Must be descriptive (e.g. 'complete order' instead of 'click here'), must indicate the action of the link or button ('this will open in a new window')
+
+<br><br>![alt text](content/images/a11ylinkalt.png "example of accessible table")
+<br>Example of an accessible link
+<br>
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Design - Modals and Pop-Up Windows
+- Modals should be developed for information purposes and not have complex interactive elements.  Modals should limit content if possible for simplicity 
+- Pages with lengthy content should have clear markers indicating the scope of the content and provide the ability to jump to key points including the beginning and end
+
+
+<br><br>![alt text](content/images/a11ymodal.jpg "example of accessible modal window with link to top of content")
+<br>Example of an accessible modal window with link to top of page
+<br>
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Design - Templates
+
+- Using the above items, it is recommended a standard template for your web projects be developed
+- Demo:  Accessible forms-template in angular 2 
+
+---
+<!-- .slide: id="a11y" -->
+## Accessible Development
+
+To acheive accessibility set out in design, developers can use the following tips, do's, and don'ts.
+
+General Considerations
```

Comment:
delete "To achieve..." lines and "General Considerations" line - these are slides, so it's OK to go straight into the bullet points.